projects/ion/src/lib/message/message.component.html

```diff
@@ -0,0 +1,4 @@
+<div data-testid="ion-message" [class]="'ion-message ' + type">
```

Comment:
You also don't need the class for the type, you can remove it from here since it's not being used for css purposes. In the tests you can check if the element has the attr (with fixture)
```suggestion
<div data-testid="ion-message" class="ion-message">
```